src/content/whats-new/2021/05/rss.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "RSS for docs so you can automate all the Slack bots"
+summary: "Keep track of what's new with RSS feeds for What's New (and security bulletins)"
+releaseDate: '2021-05-21'
+---
+Have you heard? RSS is back! Our docs have long included RSS feeds for each of our agents. Lately we've seen lots of requests to add RSS for What's New and Security Bulletins, so we've done just that! Now you can subscribe to all the newness New Relic has to offer.
+
+RSS is great in your feed reader, of course, but it also makes it easy to automatically keep up with New Relic in other places. For example, [Slack's built in RSS app](https://slack.com/help/articles/218688467-Add-RSS-feeds-to-Slack) can automatically post to a channel whenever a new agent release drops, or you could use Zapier for all manner of RSS-based automation. (Want to turn your smart lightbulbs green every time we release a new Node.js agent? Have at it!)
```

Comment:
That final "Have at it!" is a little weird. How about "Go for it!"'?

Reply:
Good call!